recipe/meta.yaml

```diff
@@ -13,19 +13,19 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 1
+  number: 0
```

Comment:
Would bump the build number as well too as bad packages may have leaked out from the old PR ( https://github.com/conda-forge/sklearn-contrib-lightning-feedstock/pull/4 ).

Reply:
what should I put here? I just want to update the version with the latest release ...